### PR TITLE
Enabled 'un-star' message option in the right tab bar

### PR DESCRIPTION
**Pull request by bgomelsky**

### CODE REVIEW

File: packages/rocketchat-message-star/client/views/starredMessages.coffee [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 
 	messages: ->
-		return StarredMessage.find { rid: Session.get 'openedRoom' }, { sort: { ts: -1 } }
+		return StarredMessage.find { rid: this.rid }, { sort: { ts: -1 } }
 
 	notReadySubscription: ->
 		return 'notready' unless Template.instance().subscriptionsReady()
 
 Template.starredMessages.onCreated ->
-	this.autorun =>
```
Comment (geekgonecrazy) [inline packages/rocketchat-message-star/client/views/starredMessages.coffee]: Curious what is the reasoning for removing this?

Comment (bgomelsky) [inline packages/rocketchat-message-star/client/views/starredMessages.coffee]: That change wasn't supposed to be part of this PR. I will update it and resubmit.

**Comment by geekgonecrazy** [comment]
@bgomelsky can you include a screenshot of what its supposed to look like?

**Comment by bgomelsky** [comment]
un-star message from right tab bar
Originally right tab was missing the message cog.

**Comment by gmsecrieru** [comment]
Cool, thanks!!

**Comment by geekgonecrazy** [comment]
This looks good.  From my quick test too this works.

**Comment by marceloschmidt** [comment]
I was actually checking this one yesterday, and it didn't look OK, but  not
really the un-star part of it. With the change you're also able to edit
your  own starred messages, and if they are not loaded on screen, it
breaks. I'm working on fixing it :)

Em qui, 22 de out de 2015 01:50, Aaron Ogle <notifications@github.com>
escreveu:

> Merged #1178 <https://github.com/RocketChat/Rocket.Chat/pull/1178>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RocketChat/Rocket.Chat/pull/1178#event-442118308>.
>
-- 

Marcelo Schmidt

**Comment by geekgonecrazy** [comment]
Oh!  Yikes

**Comment by marceloschmidt** [comment]
No need to revert it ;) I'll work on the master branch, as it's probably a quick fix. Thanks.

**Comment by marceloschmidt** [comment]
Solution was to remove edit and delete action buttons from starred messages list... Maybe not the best solution, but hey, it's starred messages list, not a quick access list, huh? You can unstar from there, but if you need to edit or delete, I guess you need to know the context, and for that you'll have to have the message on screen.

**Comment by geekgonecrazy** [comment]
@marceloschmidt that works.  :smile: once we have the ability to jump to a message then editing / deleting will be easy.

**Pull request merged**